CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # CHANGELOG
 
+## v1.2.4
```

Comment:
Why not v1.3, since the SPG has include the support of protocol v1.3?

Reply:
We’ll be making a separate release for v1.3 in a future PR. For now, we created a release from the `release-v1.2.4` branch for v1.2.4, which is designed to work with the protocol core v1.2.3 release. This PR is to merge the changes from v1.2.4 into the main branch, so we have a complete history of changes on the main branch.